### PR TITLE
Add detection for Cantonese (yue)

### PR DESCRIPTION
**Pull request by rylans**
Added detection for cantonese using most common characters that are not found commonly in Mandarin. Added a test.

### CODE REVIEW

File: lang.go [hunk 1]
```diff
@@ -16,6 +16,7 @@ const (
 	Ceb
 	Ces
 	Cmn
+	Yue
```
Comment (codemoomba) [inline lang.go]: The languages are in order alphabetically (in English). This line should go above `Zul`.

File: lang.go [hunk 2]
```diff
@@ -115,6 +116,8 @@ func CodeToLang(code string) Lang {
 		return Ces
 	case "cmn":
 		return Cmn
+	case "yue":
```
Comment (codemoomba) [inline lang.go]: The languages are in order alphabetically (in English). These lines should go above `case "zul":`.

File: lang.go [hunk 3]
```diff
@@ -291,6 +294,8 @@ func LangToStringShort(lang Lang) string {
 		return "cs"
 	case Cmn:
 		return "zh" // No iso 639-1, but http://www.loc.gov/standards/iso639-2/faq.html#24
+	case Yue:
```
Comment (codemoomba) [inline lang.go]: The languages are in order alphabetically (in English). These lines should go above `case Zul:`.

File: lang.go [hunk 4]
```diff
@@ -465,6 +470,8 @@ func LangToString(lang Lang) string {
 		return "ces"
 	case Cmn:
 		return "cmn"
+	case Yue:
```
Comment (codemoomba) [inline lang.go]: The languages are in order alphabetically (in English). These lines should go above `case Zul:`.

File: lang.go [hunk 5]
```diff
@@ -627,6 +634,7 @@ var Langs = map[Lang]string{
 	Ceb: "Cebuano",
 	Ces: "Czech",
 	Cmn: "Mandarin",
+	Yue: "Cantonese",
```
Comment (codemoomba) [inline lang.go]: The languages are in order alphabetically (in English). This line should go above `Zul: "Zulu"`.

**Comment by rylans** [comment]
Thanks for the review. But I don't think this change fits in with this project upon second thought.

**Pull request closed without merging**